src/pages-conditional/dapps.js

```diff
@@ -0,0 +1,1150 @@
+  }
+`
+
+const H3 = styled.h3`
+  font-size: 20px;
+  font-weight: 700;
+  margin-bottom: 0.5rem;
+  a {
+    display: none;
+  }
+`
+
+const CenterText = styled.p`
+  text-align: center;
+  max-width: 800px;
+`
+
+const LeftColumn = styled.div`
+  margin-right: 2rem;
+  width: 100%;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin: auto 0rem;
+  }
+`
+
+const RightColumn = styled.div`
+  margin-left: 2rem;
+  width: 100%;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin: auto 0rem;
+  }
+`
+
+const About = styled.div`
+  margin-top: 3rem;
+`
+
+const Box = styled.div`
+  text-align: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 3rem;
+`
+
+const paths = [
+  {
+    emoji: ":open_lock:",
+    title: "Open access",
+    description:
+      "Finance services running on Ethereum have no sign up requirements. If you have funds and an internet connection, you’re good to go.",
+  },
+  {
+    emoji: ":bank:",
+    title: "A new token economy",
+    description:
+      "There’s a whole world of tokens that you can interact with across these financial products. People are building new tokens on top of Ethereum all the time.",
+  },
+  {
+    emoji: ":scales:",
+    title: "Stablecoins",
+    description:
+      "Teams have built stablecoins – a less volatile cryptocurrency. These allow you to experiment and use crypto without the risk and uncertainty.",
+  },
+  {
+    emoji: ":chart_with_upwards_trend:",
+    title: "Crypto can earn interest",
+    description:
+      "Funding borrowing pools, borrowing, etc.. There are incentives for providing liquidity to markets. This keeps the supply and demand balanced and strong...",
+  },
+]
+const artsCollectibles = [
+  {
+    emoji: ":open_lock:",
+    title: "Non-fungible tokens",
+    description:
+      "When art is tokenised on Ethereum, ownership can be proved for all to see. You can trace the artwork's journey from creation to its current holder. This prevents forgeries.",
+  },
+  {
+    emoji: ":bank:",
+    title: "Peer-to-peer streaming",
+    description: "Something about music streaming.",
+  },
+  {
+    emoji: ":bank:",
+    title: "Collectibles go with you",
+    description:
+      "Tokenised collectibles are tied to your address. That means they're tied to you not the platform you got it from. This means you can sell things like in-game items on any Ethereum marketplace, not just in the game itself.",
+  },
+  {
+    emoji: ":bank:",
+    title: "Infrastructure already in place",
+    description:
+      "The tools and products already exist for you to tokenise your stuff and sell it!",
+  },
+]
+
+const DappsPage = ({ data }) => {
+  const [isFinance, setIsFinance] = useState(true)
+  const [isTechnology, setIsTechnology] = useState(false)
+  const [isCollectibles, setIsCollectibles] = useState(false)
+  const [isGaming, setIsGaming] = useState(false)
+
+  const lending = [
```

Comment:
@ryancreatescopy I think another financial category that's worth including is insurance, e.g. Nexus Mutual.